benchopt/cli/tests/test_cli.py

```diff
@@ -361,7 +361,7 @@ def test_caching(self, n_rep):
             # Check that the computation caching is working properly.
             run_cmd = (
                 f"{bench.benchmark_dir} -d test-dataset -s test-solver "
-                f"-n 1 -r {n_rep} --no-plot"
+                f"-n 1 -r {n_rep} --no-plot --seed 0"
```

Comment:
This seeds are probably not needed anymore as the seed is only used for caching if the benchmark uses the seed, which is not the case by default?